src/main/java/technical/SaveFile.java

```diff
@@ -0,0 +1,104 @@
+package technical;
+/**
+ * The class which organises the save file.
+ * @author Nicholas Patrick
+ */
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+public class SaveFile {
+  private static final String SAVE_FILE_NAME = ".data.txt";
+  private static final String TEMPORARY_FILE_NAME = ".data_temp.txt";
+  private static final File saveFile = new File(SAVE_FILE_NAME);
```

Comment:
since final variables don't change, maybe the naming should follow naming of constants, and should be in uppercase.